pallets/valve/src/lib.rs

```diff
@@ -53,6 +54,10 @@ pub mod pallet {
 		ValveClosed,
 		/// The chain returned to its normal operating state.
 		ValveOpen,
+		/// All the pallet's actions stoped.
+		PalletTapped { pallet_name_bytes: Vec<u8> },
+		/// All the pallet's actions resumed.
+		PalletUntapped { pallet_name_bytes: Vec<u8> },
```

Comment:
`PalletGateClosed`: All pallet's actions stopped.
`PalletGateOpen`: All ... opened.

Inspo https://www.robinsonsplumbingservice.com/knowing-where-youre-main-water-shut-off-valve-is-location/